feat: remove CTA

asadali145 opened the pull request:
### Relevant Tickets
https://github.com/openedx/frontend-app-ora-grading/issues/360

### Description:

The CTA was added two years ago. It is enabled by default, and the notification is always visible. This PR removes the CTA. If we still need it, we can put it behind a flag.

### Screen Shots:
Before:
![Screenshot 2024-08-20 at 1 26 05 PM](https://github.com/user-attachments/assets/99577350-76d4-4afd-9648-edff1433c5fa)

After:
<img width="1739" alt="Screenshot 2024-10-02 at 12 33 20 PM" src="https://github.com/user-attachments/assets/8403d9df-675f-4c02-8eb8-701b6c99c8df">

openedx-webhooks commented:
Thanks for the pull request, @asadali145!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/openedx-unmaintained`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

asadali145 commented:
@jansenk Would you be able to review this PR?

asadali145 commented:
@brian-smith-tcril Would you be able to review this PR?

brian-smith-tcril commented:
Looking at this PR raises a few questions for me:
* Do we want to remove the CTA?
* Do we want to completely remove the notification banner if “preferences center” would be a broken link?
  * This effectively turns `ACCOUNT_SETTINGS_URL` into a "show/hide" notification banner toggle which seems less than ideal.
* Do we need the notification banner *at all*?

My feeling is both the CTA and notification banner should just be removed completely. This would require going through [the deprecation process](https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0021-proc-deprecation.html).

Would fully removing both the notification banner and the CTA solve the problem you're trying to solve with this PR?

brian-smith-tcril commented:
> Do you want me to start the depreciation process for these banners?

That would be wonderful @asadali145! Thank you!

asadali145 commented:
@brian-smith-tcril I have created this DEPR ticket https://github.com/openedx/frontend-app-ora-grading/issues/360. Do you have any suggestions?

awais-ansari commented on src/containers/NotificationsBanner/index.jsx:
```diff
@@ -6,7 +7,8 @@ import { PageBanner, Hyperlink } from '@openedx/paragon';
 import messages from './messages';
 
-export const NotificationsBanner = () => (
+// eslint-disable-next-line no-confusing-arrow
```
Please follow the eslint recommendations. Other than PR LGTM.

asadali145 commented:
Thanks @brian-smith-tcril and @awais-ansari. I have updated this PR to remove the CTA. I'll create a new PR to update the notifications banner. I have updated the DEPR ticket as well.

asadali145 commented:
@brian-smith-tcril Just a reminder that this PR needs review along with https://github.com/openedx/frontend-app-ora-grading/pull/362 as discussed to created separate PRs.

brian-smith-tcril reviewed (approved):
Let's merge it! (context on DEPR approval https://github.com/openedx/frontend-app-ora-grading/issues/360#issuecomment-2452307704)

brian-smith-tcril commented:
@asadali145 sorry I missed this! It looks like it needs another rebase. I've communicated my intention to merge, so as soon as it is rebased I'll hit the button.

asadali145 commented:
@brian-smith-tcril This is rebased. You can merge.
